### PR TITLE
Added action for checking commit message style

### PR DESCRIPTION
**Pull request by telday**
### What does this PR do?
Added new action which will check commit style before allowing a pull request to be merged in.

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -14,3 +17,17 @@ jobs:
...
+    runs-on: ubuntu-latest
+    name: Enforce commit style
+
+    steps:
+    - name: Checkout the repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Check commit style
+      uses: telday/enforce-commit-style@v1
```
Comment (orenbm) [inline .github/workflows/main.yml:31]: do we want to move this to the `cyberark` org?

Comment (orenbm) [reply .github/workflows/main.yml:31]: oh i see that `https://github.com/telday/enforce-commit-style` redirects to `https://github.com/cyberark/enforce-commit-style`. in that case can we use the `cyberark` one?

**Comment by orenbm** [comment]
Thanks for getting this @telday !

I added [some issues](https://github.com/cyberark/enforce-commit-style/issues) to the GH action project that i will try to get to soon. i would like to learn how to write GH actions so it can be a good experience for me. feel free to comment on the issues if you have any thoughts.

File: .github/workflows/main.yml (hunk 1 above)
Comment (telday) [reply .github/workflows/main.yml]: My bad, also going to need to fix that on the main repo

**Review by orenbm — APPROVED** [review]

**Pull request merged**